shared/teams/team/rows/channel-row/channel.tsx

```diff
@@ -30,6 +30,7 @@ const ChannelRow = (props: ChannelRowProps) => {
   const activityLevel = Container.useSelector(
     state => state.teams.activityLevels.channels.get(channel.conversationIDKey) || 'none'
   )
+  const channelMeta = useChannelMeta(teamID, conversationIDKey)
```

Comment:
shouldn't need this, channelname and description are on the `channel` obj above

Reply:
:o sorry I missed this